pkg/cmd/server/start/start_api.go

```diff
@@ -92,8 +92,10 @@ func NewCommandStartMasterAPI(name, basename string, out io.Writer) (*cobra.Comm
 		config.EtcdConfig = nil
 		if config.KubernetesMasterConfig != nil {
 			if masterAddr.Provided {
-				glog.V(2).Infof("Using a master address override %q", masterAddr.Host)
-				config.KubernetesMasterConfig.MasterIP = masterAddr.Host
+				if ip := net.ParseIP(masterAddr.Host); ip != nil {
```

Comment:
yeah, actually, I want to parse and ignore if it's not an IP... otherwise, this breaks, which feels stupid:
```
openshift start master --master=https://localhost:8443 --write-config=foo
openshift start master --master=https://localhost:8443 --config=foo/master-config.yaml
```